index.html

```diff
@@ -6920,7 +6920,7 @@ <h2>Decoder gamma
 calibration measurements are available for the particular <a>CRT</a>
 used.</p>
 
-<p>Many modern frame buffers have lookup tables that are used to
+<p>Many modern <a>frame buffer</a>s have lookup tables that are used to
```

Comment:
I think ReSpec is okay with plurals. If you haven't tried, can you see if including the 's' is fine?

Reply:
Thanks for the catch!

Reply:
```suggestion
<p>Many modern <a>frame buffers</a> have lookup tables that are used to
```